--- src/app/main.cpp.old	2009-03-18 20:15:43.000000000 -0300
+++ src/app/main.cpp	2009-03-18 21:00:27.000000000 -0300
@@ -179,7 +179,7 @@ static inline QStringList getPluginPaths
     // 1) "plugins" (Win/Linux)
     QString pluginPath = rootDirPath;
     pluginPath += QDir::separator();
-    pluginPath += QLatin1String("lib");
+    pluginPath += QLatin1String("lib64");
     pluginPath += QDir::separator();
     pluginPath += QLatin1String("qtcreator");
     pluginPath += QDir::separator();
