diff -u -r a/doc/api/qtcreator-documentation.qdoc b/doc/api/qtcreator-documentation.qdoc
--- a/doc/api/qtcreator-documentation.qdoc	2015-06-26 15:14:57.000000000 -0300
+++ b/doc/api/qtcreator-documentation.qdoc	2015-06-27 08:06:57.816782261 -0300
@@ -277,7 +277,7 @@
     The \QC Manual HTML files are generated in the \c {doc/html} directory.
     The Extending \QC Manual files are generated in the
     \c {doc/html-dev} directory. The help files (\c {.qch}) are generated in the
-    \c {share/doc/qtcreator} directory in the \QC build directory on Windows and
+    \c {share/doc/qt-creator} directory in the \QC build directory on Windows and
     Linux, and in the \c {bin/Qt Creator.app/Contents/Resources/app} directory
     on OS X. You can view the HTML files in a browser and the help files in
     the \QC \uicontrol Help mode. For more information about adding the help
diff -u -r a/doc/doc.pri b/doc/doc.pri
--- a/doc/doc.pri	2015-06-26 15:14:57.000000000 -0300
+++ b/doc/doc.pri	2015-06-27 08:09:05.019986300 -0300
@@ -68,12 +68,12 @@
 
 !macx {
     inst_qch_docs.files = $$QCH_FILE
-    inst_qch_docs.path = $$QTC_PREFIX/share/doc/qtcreator
+    inst_qch_docs.path = $$QTC_PREFIX/share/doc/qt-creator
     inst_qch_docs.CONFIG += no_check_exist no_default_install
     INSTALLS += inst_qch_docs
 
     inst_dev_qch_docs.files = $$DEV_QCH_FILE
-    inst_dev_qch_docs.path = $$QTC_PREFIX/share/doc/qtcreator
+    inst_dev_qch_docs.path = $$QTC_PREFIX/share/doc/qt-creator
     inst_dev_qch_docs.CONFIG += no_check_exist no_default_install
     INSTALLS += inst_dev_qch_docs
 
diff -u -r a/qtcreator.pri b/qtcreator.pri
--- a/qtcreator.pri	2015-06-26 15:14:57.000000000 -0300
+++ b/qtcreator.pri	2015-06-27 08:09:40.862081009 -0300
@@ -90,7 +90,7 @@
     IDE_PLUGIN_PATH  = $$IDE_LIBRARY_PATH/plugins
     IDE_LIBEXEC_PATH = $$IDE_APP_PATH # FIXME
     IDE_DATA_PATH    = $$IDE_BUILD_TREE/share/qtcreator
-    IDE_DOC_PATH     = $$IDE_BUILD_TREE/share/doc/qtcreator
+    IDE_DOC_PATH     = $$IDE_BUILD_TREE/share/doc/qt-creator
     IDE_BIN_PATH     = $$IDE_APP_PATH
     !isEqual(IDE_SOURCE_TREE, $$IDE_BUILD_TREE):copydata = 1
 }
diff -u -r a/qtcreator.qbs b/qtcreator.qbs
--- a/qtcreator.qbs	2015-06-26 15:14:57.000000000 -0300
+++ b/qtcreator.qbs	2015-06-27 08:10:22.472226518 -0300
@@ -45,7 +45,7 @@
             : ide_app_path
     property string ide_doc_path: qbs.targetOS.contains("osx")
             ? ide_data_path + "/doc"
-            : "share/doc/qtcreator"
+            : "share/doc/qt-creator"
     property string ide_bin_path: qbs.targetOS.contains("osx")
             ? ide_app_target + ".app/Contents/MacOS"
             : ide_app_path
diff -u -r a/src/plugins/coreplugin/icore.cpp b/src/plugins/coreplugin/icore.cpp
--- a/src/plugins/coreplugin/icore.cpp	2015-06-26 15:14:57.000000000 -0300
+++ b/src/plugins/coreplugin/icore.cpp	2015-06-27 08:11:31.396467985 -0300
@@ -400,7 +400,7 @@
 QString ICore::resourcePath()
 {
     const QString sharePath = QLatin1String(Utils::HostOsInfo::isMacHost()
-                                            ? "/../Resources" : "/../share/qtcreator");
+                                            ? "/../Resources" : "/../share/qt-creator");
     return QDir::cleanPath(QCoreApplication::applicationDirPath() + sharePath);
 }
 
