diff -rupN qt-creator-opensource-src-4.7.1/src/tools/clangbackend/source/clangbackend_global.h qt-creator-opensource-src-4.7.1-new/src/tools/clangbackend/source/clangbackend_global.h
--- qt-creator-opensource-src-4.7.1/src/tools/clangbackend/source/clangbackend_global.h	2018-09-18 10:30:26.000000000 +0200
+++ qt-creator-opensource-src-4.7.1-new/src/tools/clangbackend/source/clangbackend_global.h	2018-09-20 23:34:47.601417022 +0200
@@ -38,22 +38,22 @@ enum class PreferredTranslationUnit
 
 // CLANG-UPGRADE-CHECK: Remove IS_PRETTY_DECL_SUPPORTED once we require clang >= 7.0
 #if defined(CINDEX_VERSION_HAS_PRETTYDECL_BACKPORTED) || CINDEX_VERSION_MINOR >= 46
-#  define IS_PRETTY_DECL_SUPPORTED
+// #  define IS_PRETTY_DECL_SUPPORTED
 #endif
 
 // CLANG-UPGRADE-CHECK: Remove IS_INVALIDDECL_SUPPORTED once we require clang >= 7.0
 #if defined(CINDEX_VERSION_HAS_ISINVALIDECL_BACKPORTED) || CINDEX_VERSION_MINOR >= 46
-#  define IS_INVALIDDECL_SUPPORTED
+// #  define IS_INVALIDDECL_SUPPORTED
 #endif
 
 // CLANG-UPGRADE-CHECK: Remove IS_LIMITSKIPFUNCTIONBODIESTOPREAMBLE_SUPPORTED once we require clang >= 7.0
 #if defined(CINDEX_VERSION_HAS_LIMITSKIPFUNCTIONBODIESTOPREAMBLE_BACKPORTED) || CINDEX_VERSION_MINOR >= 46
-#  define IS_LIMITSKIPFUNCTIONBODIESTOPREAMBLE_SUPPORTED
+// #  define IS_LIMITSKIPFUNCTIONBODIESTOPREAMBLE_SUPPORTED
 #endif
 
 // CLANG-UPGRADE-CHECK: Remove IS_SKIPWARNINGSFROMINCLUDEDFILES_SUPPORTED once we require clang >= 7.0
 #if defined(CINDEX_VERSION_HAS_SKIPWARNINGSFROMINCLUDEDFILES_BACKPORTED) || CINDEX_VERSION_MINOR >= 46
-#  define IS_SKIPWARNINGSFROMINCLUDEDFILES_SUPPORTED
+// #  define IS_SKIPWARNINGSFROMINCLUDEDFILES_SUPPORTED
 #endif
 
 } // namespace ClangBackEnd
