diff -up qt-creator-2.4.1-src/src/tools/valgrindfake/outputgenerator.cpp.include-unistd-valgrindfake qt-creator-2.4.1-src/src/tools/valgrindfake/outputgenerator.cpp
--- qt-creator-2.4.1-src/src/tools/valgrindfake/outputgenerator.cpp.include-unistd-valgrindfake	2012-01-24 15:37:13.000000000 +0100
+++ qt-creator-2.4.1-src/src/tools/valgrindfake/outputgenerator.cpp	2012-02-01 14:17:53.436607829 +0100
@@ -41,6 +41,8 @@
 #include <QStringList>
 #include <QDebug>
 
+#include <unistd.h>
+
 using namespace Valgrind::Fake;
 
 OutputGenerator::OutputGenerator(QAbstractSocket *output, QIODevice *input) :
