diff -rupN qt-creator-opensource-src-4.8.0-beta1/src/plugins/cmakeprojectmanager/cmakekitinformation.cpp qt-creator-opensource-src-4.8.0-beta1-new/src/plugins/cmakeprojectmanager/cmakekitinformation.cpp
--- qt-creator-opensource-src-4.8.0-beta1/src/plugins/cmakeprojectmanager/cmakekitinformation.cpp	2018-10-09 14:54:47.000000000 +0200
+++ qt-creator-opensource-src-4.8.0-beta1-new/src/plugins/cmakeprojectmanager/cmakekitinformation.cpp	2018-10-11 11:30:53.581801781 +0200
@@ -325,7 +325,7 @@ QVariant CMakeGeneratorKitInformation::d
     if (it != known.constEnd()) {
         Utils::Environment env = Utils::Environment::systemEnvironment();
         k->addToEnvironment(env);
-        const Utils::FileName ninjaExec = env.searchInPath(QLatin1String("ninja"));
+        const Utils::FileName ninjaExec = env.searchInPath(QLatin1String("ninja-build"));
         if (!ninjaExec.isEmpty())
             return GeneratorInfo({QString("Ninja"), extraGenerator, QString(), QString()}).toVariant();
     }
