From 8f2d0094fd4e8ec929d804f79f1db3136f016b27 Mon Sep 17 00:00:00 2001
From: dt <qtc-committer@nokia.com>
Date: Mon, 29 Jun 2009 12:26:53 +0200
Subject: [PATCH] Fix qtcreatorwidgets.pro to use the same logic as qtcreator.pri

---
 src/tools/qtcreatorwidgets/qtcreatorwidgets.pro |   11 ++++++-----
 1 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/src/tools/qtcreatorwidgets/qtcreatorwidgets.pro b/src/tools/qtcreatorwidgets/qtcreatorwidgets.pro
index ad78021..1668c4a 100644
--- a/src/tools/qtcreatorwidgets/qtcreatorwidgets.pro
+++ b/src/tools/qtcreatorwidgets/qtcreatorwidgets.pro
@@ -9,16 +9,17 @@ SOURCES     = customwidgets.cpp
 
 # Link against the qtcreator utils lib
 
+isEmpty(IDE_LIBRARY_BASENAME) {
+    IDE_LIBRARY_BASENAME = lib
+}
+
 linux-* {
   # form abs path to qtcreator lib dir
   QTC_LIBS=$$dirname(PWD)
   QTC_LIBS=$$dirname(QTC_LIBS)
   QTC_LIBS=$$dirname(QTC_LIBS)
-  linux-*64 {
-    QTC_LIBS=$$QTC_LIBS/lib64/qtcreator
-  } else {
-    QTC_LIBS=$$QTC_LIBS/lib/qtcreator
-  }
+  QTC_LIBS=$$QTC_LIBS/$$IDE_LIBRARY_BASENAME/qtcreator
+  QTC_LIBS=$$QTC_LIBS/$$IDE_LIBRARY_BASENAME/qtcreator
   QMAKE_RPATHDIR *= $$QTC_LIBS
 }
 
-- 
1.6.1


