diff -rupN qt-creator-opensource-src-4.7.0-beta1/dist/org.qt-project.qtcreator.desktop qt-creator-opensource-src-4.7.0-beta1-new/dist/org.qt-project.qtcreator.desktop
--- qt-creator-opensource-src-4.7.0-beta1/dist/org.qt-project.qtcreator.desktop	2018-06-04 06:52:27.000000000 +0200
+++ qt-creator-opensource-src-4.7.0-beta1-new/dist/org.qt-project.qtcreator.desktop	2018-06-07 10:31:39.773923625 +0200
@@ -7,5 +7,5 @@ X-KDE-StartupNotify=true
 Icon=QtProject-qtcreator
 Terminal=false
 Categories=Development;IDE;Qt;
-MimeType= text/x-c++src;text/x-c++hdr;text/x-xsrc;application/x-designer;application/vnd.qt.qmakeprofile;application/vnd.qt.xml.resource;
+MimeType=text/x-c++src;text/x-c++hdr;text/x-xsrc;application/x-designer;application/vnd.qt.qmakeprofile;application/vnd.qt.xml.resource;
 
