diff -rupN qt-creator-opensource-src-3.3.1/doc/api/qtcreator-documentation.qdoc qt-creator-opensource-src-3.3.1-new/doc/api/qtcreator-documentation.qdoc
--- qt-creator-opensource-src-3.3.1/doc/api/qtcreator-documentation.qdoc	2015-02-19 11:16:46.000000000 +0100
+++ qt-creator-opensource-src-3.3.1-new/doc/api/qtcreator-documentation.qdoc	2015-02-24 14:30:36.319452909 +0100
@@ -294,7 +294,7 @@
     The \QC Manual HTML files are generated in the \c {doc/html} directory.
     The Extending \QC Manual files are generated in the
     \c {doc/html-dev} directory. The help files (.qch) are generated in the
-    \c {share/doc/qtcreator} directory in the \QC build directory on Windows and
+    \c {share/doc/qt-creator} directory in the \QC build directory on Windows and
     Linux, and in the \c {bin/Qt Creator.app/Contents/Resources/app} directory
     on OS X. You can view the HTML files in a browser and the help files in
     the \QC \uicontrol Help mode. For more information about adding the help files to
diff -rupN qt-creator-opensource-src-3.3.1/doc/doc.pri qt-creator-opensource-src-3.3.1-new/doc/doc.pri
--- qt-creator-opensource-src-3.3.1/doc/doc.pri	2015-02-19 11:16:46.000000000 +0100
+++ qt-creator-opensource-src-3.3.1-new/doc/doc.pri	2015-02-24 14:30:36.319452909 +0100
@@ -68,12 +68,12 @@ dev_qch_docs.depends += dev_html_docs
 
 !macx {
     inst_qch_docs.files = $$QCH_FILE
-    inst_qch_docs.path = $$QTC_PREFIX/share/doc/qtcreator
+    inst_qch_docs.path = $$QTC_PREFIX/share/doc/qt-creator
     inst_qch_docs.CONFIG += no_check_exist no_default_install
     INSTALLS += inst_qch_docs
 
     inst_dev_qch_docs.files = $$DEV_QCH_FILE
-    inst_dev_qch_docs.path = $$QTC_PREFIX/share/doc/qtcreator
+    inst_dev_qch_docs.path = $$QTC_PREFIX/share/doc/qt-creator
     inst_dev_qch_docs.CONFIG += no_check_exist no_default_install
     INSTALLS += inst_dev_qch_docs
 
diff -rupN qt-creator-opensource-src-3.3.1/qtcreator.pri qt-creator-opensource-src-3.3.1-new/qtcreator.pri
--- qt-creator-opensource-src-3.3.1/qtcreator.pri	2015-02-19 11:16:46.000000000 +0100
+++ qt-creator-opensource-src-3.3.1-new/qtcreator.pri	2015-02-24 14:30:36.320452927 +0100
@@ -90,7 +90,7 @@ macx {
     IDE_PLUGIN_PATH  = $$IDE_LIBRARY_PATH/plugins
     IDE_LIBEXEC_PATH = $$IDE_APP_PATH # FIXME
     IDE_DATA_PATH    = $$IDE_BUILD_TREE/share/qtcreator
-    IDE_DOC_PATH     = $$IDE_BUILD_TREE/share/doc/qtcreator
+    IDE_DOC_PATH     = $$IDE_BUILD_TREE/share/doc/qt-creator
     IDE_BIN_PATH     = $$IDE_APP_PATH
     !isEqual(IDE_SOURCE_TREE, $$IDE_BUILD_TREE):copydata = 1
 }
diff -rupN qt-creator-opensource-src-3.3.1/qtcreator.qbs qt-creator-opensource-src-3.3.1-new/qtcreator.qbs
--- qt-creator-opensource-src-3.3.1/qtcreator.qbs	2015-02-19 11:16:46.000000000 +0100
+++ qt-creator-opensource-src-3.3.1-new/qtcreator.qbs	2015-02-24 14:30:36.320452927 +0100
@@ -42,7 +42,7 @@ Project {
             : ide_app_path
     property string ide_doc_path: qbs.targetOS.contains("osx")
             ? ide_data_path + "/doc"
-            : "share/doc/qtcreator"
+            : "share/doc/qt-creator"
     property string ide_bin_path: qbs.targetOS.contains("osx")
             ? ide_app_target + ".app/Contents/MacOS"
             : ide_app_path
diff -rupN qt-creator-opensource-src-3.3.1/src/plugins/coreplugin/icore.cpp qt-creator-opensource-src-3.3.1-new/src/plugins/coreplugin/icore.cpp
--- qt-creator-opensource-src-3.3.1/src/plugins/coreplugin/icore.cpp	2015-02-19 11:16:46.000000000 +0100
+++ qt-creator-opensource-src-3.3.1-new/src/plugins/coreplugin/icore.cpp	2015-02-24 14:30:36.321452946 +0100
@@ -422,7 +422,7 @@ QString ICore::userResourcePath()
 QString ICore::documentationPath()
 {
     const QString docPath = QLatin1String(Utils::HostOsInfo::isMacHost()
-                                            ? "/../Resources/doc" : "/../share/doc/qtcreator");
+                                            ? "/../Resources/doc" : "/../share/doc/qt-creator");
     return QDir::cleanPath(QCoreApplication::applicationDirPath() + docPath);
 }
 
