diff -rupN qt-creator-opensource-src-3.1.1/src/plugins/projectexplorer/gcctoolchain.cpp qt-creator-opensource-src-3.1.1-new/src/plugins/projectexplorer/gcctoolchain.cpp
--- qt-creator-opensource-src-3.1.1/src/plugins/projectexplorer/gcctoolchain.cpp	2014-05-15 12:45:54.000000000 +0200
+++ qt-creator-opensource-src-3.1.1-new/src/plugins/projectexplorer/gcctoolchain.cpp	2014-06-22 18:27:42.569947985 +0200
@@ -428,7 +428,7 @@ ToolChain::CompilerFlags GccToolChain::c
     const QStringList allCxxflags = m_platformCodeGenFlags + cxxflags; // add only cxxflags is empty?
     foreach (const QString &flag, allCxxflags) {
         if (flag.startsWith(QLatin1String("-std="))) {
-            const QByteArray std = flag.mid(5).toAscii();
+            const QByteArray std = flag.mid(5).toLatin1();
             if (std == "c++98" || std == "c++03") {
                 flags &= ~CompilerFlags(StandardCxx11 | GnuExtensions);
             } else if (std == "gnu++98" || std == "gnu++03") {
