--- src/rpath.pri.old	2009-03-17 20:37:28.000000000 -0300
+++ src/rpath.pri	2009-03-17 20:38:08.000000000 -0300
@@ -3,7 +3,7 @@ macx {
 } else:linux-* {
     #do the rpath by hand since it's not possible to use ORIGIN in QMAKE_RPATHDIR
     # this expands to $ORIGIN (after qmake and make), it does NOT read a qmake var
-    QMAKE_RPATHDIR += \$\$ORIGIN/../lib/qtcreator
+    QMAKE_RPATHDIR += \$\$ORIGIN/../lib64/qtcreator
     IDE_PLUGIN_RPATH = $$join(QMAKE_RPATHDIR, ":")
  
     QMAKE_LFLAGS += -Wl,-z,origin \'-Wl,-rpath,$${IDE_PLUGIN_RPATH}\'
